### ISSUE TITLE
The whole page should be rewritten to include only the very minimum code ... how simple can it get?

### ISSUE THREAD

**Issue by marcoherbst** [evercam/wicklowcam#11]

**Comment by asimism** [comment]
I think we should think about this option because I see a lot of code/stuff which is not making any sense when we see the page in browser. For example there is:
1. Facebook javascript code which I never see on my screen.
2. There is big html table with full screen and comments buttons (possibly buttons) but no associated js 
3. Also on mobiles portrait view it has a strange look. (blue bars below camera view) We should consider asking a designer to suggest appropriate design for mobile portrait.
Best Regards
Asim Ismail

**Comment by marcoherbst** [comment]
Right, that's the whole point - strip it ALL out . It's not needed. That's
the point of this ticket.

We just want the full-screen video and the logos.

The video should try to fill the screen. If height is the limiting factor,
then we put the logos below the video, so they are off screen and only
visible by scrolling.  That is the ideal scenario.
Everything else that is on that page can go.

---

The purpose of this repo is to keep a model for a very simple, one page "My
Webcam" type page for any camera connected to Evercam. We want to make it
easy for someone to clone this repository and have their own webcam page,
hosted on Github. The different branches would include different
functionality, e.g. Facebook.

---

On 22 August 2015 at 13:04, asimism notifications@github.com wrote:

> I think we should think about this option because I see a lot of
> code/stuff which is not making any sense when we see the page in browser.
> For example there is:
> 1. Facebook javascript code which I never see on my screen.
> 2. There is big html table with full screen and comments button buttons
> (possibly buttons) but no associated js
> 3. Also on mobiles potrait view is has a strange look. (blue bars below
> camera view) We should consider asking a designer to suggest appropriate
> design for mobile portait.
> Best Regards
> Asim Ismail
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/evercam/wicklowcam/issues/11#issuecomment-133684146.

## 

Marco Herbst
_Evercam.io http://www.evercam.io/_ - Cameras Unleashed
Mobile: +353 86 196 0451
Skype: marco.a.herbst

**Comment by marcoherbst** [comment]
@asimism  ... do you feel this is done? if more could be done, I'll ask @affan00 ....
